app-office/texstudio/texstudio-4.8.2.ebuild

```diff
@@ -14,27 +14,18 @@ S="${WORKDIR}/${PN}-${MY_PV}"
 LICENSE="GPL-2"
 SLOT="0"
 KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~ppc64 ~x86"
-IUSE="video"
 
 DEPEND="
 	app-text/hunspell:=
-	app-text/poppler:=[qt5]
-	>=dev-libs/quazip-1.0:0=[qt5(+)]
-	dev-qt/designer:5
-	dev-qt/qtcore:5
-	dev-qt/qtconcurrent:5
-	dev-qt/qtdeclarative:5
-	dev-qt/qtgui:5
-	dev-qt/qtnetwork:5
-	dev-qt/qtprintsupport:5
-	dev-qt/qtsingleapplication[qt5(+),X]
-	dev-qt/qtsvg:5
-	dev-qt/qttest:5
-	dev-qt/qtwidgets:5
-	dev-qt/qtxml:5
+	app-text/poppler:=[qt6]
+	dev-qt/qtbase:6[concurrent,gui,network,widgets,xml]
+	dev-qt/qtdeclarative:6
+	dev-qt/qtmultimedia:6
+	dev-qt/qtsingleapplication[X]
```

Comment:
That's Qt5-based, so unlikely to do anything for this package. *Maybe* it needs `dev-libs/kdsingleapplication`? Of course that requires checking the build system.

Reply:
> That's Qt5-based, so unlikely to do anything for this package.

It appears to me that they still use it even with the Qt6 build.

https://github.com/texstudio-org/texstudio/blob/a1f8119a478871c7af0dfcb690963d764da03c0b/texstudio.pro#L107

And we have a sed in place that seems to unbundle qtsingleapplication at https://github.com/gentoo/gentoo/blob/b9c2f46e265766151302d139c4e86d79eecb030d/app-office/texstudio/texstudio-4.8.1.ebuild#L60-L61

This is basically why I came to #gentoo-qt asking for help. Qt is not my area of expertise. Nor is qmake and its `.pro` files and how (if) they interact with cmake.

So if someone could shed some light into the situation, have a look and assist with the bump that would be greatly appreciated. Otherwise, I have to dig into this, if I find the time doing so.

Reply:
Upstream was bundling qtsingleapplication. The `sed` was undoing that, as we should use system packages wherever possible.

To their credit, they ported their bundled lib to Qt6, even if it was completely unnecessary since kdsingleapplication exists. Maybe someone should break it to them. Using bundled libs caused more work, expectedly.

Reply:
also, 4.8.3 released just a few hours ago. :)

Reply:
> Upstream was bundling qtsingleapplication. The `sed` was undoing that, as we should use system packages wherever possible.

I know, the problem is different. I mentioned it in #-qt but forgot to write it in the PR. To quote myself:

```
11:55:19      +Flow | anyhow, the build fails because the qtsingleapplication include path is not there and I wonder if it should, given that its qt5
```